data/src/main/java/com/mashup/alcoholfree/data/dto/remote/response/DrinkingLimitResponse.kt

```diff
@@ -0,0 +1,18 @@
+package com.mashup.alcoholfree.data.dto.remote.response
+
+import com.google.gson.annotations.SerializedName
+import com.mashup.alcoholfree.domain.model.DrinkingLimit
+
+data class DrinkingLimitResponse(
+    @SerializedName("glass")
+    val glass: Int,
+    @SerializedName("type")
+    val type: String,
+) {
+    fun toDomainModel(): DrinkingLimit {
```

Comment:
```suggestion
    fun toDomainModel(): Drink {
```

DrinkingLimit는 기존에 사용하고있던 도메인 모델을 이용하는게 의미상 더 정확한거같아!